[BUG] [ERROR] Failed to check persistent ignore list: DrizzleQueryError: Failed query: select "nodeNum" from "ignored_nodes" where "ignored_nodes"."nodeNum" = $1

**Describe the bug**
Something has the incorrect type or integer set

**To Reproduce**
Fresh install

**Expected behavior**
```bash
Feb 21 02:14:50 meshtest meshmonitor[16467]: [ERROR] Failed to check persistent ignore list: DrizzleQueryError: Failed query: select "nodeNum" from "ignored_nodes" where "ignored_nodes"."nodeNum" = $1
Feb 21 02:14:50 meshtest meshmonitor[16467]: params: 2733697270
Feb 21 02:14:50 meshtest meshmonitor[16467]:     at NodePgPreparedQuery.queryWithCache (file:///opt/meshmonitor/node_modules/drizzle-orm/pg-core/session.js:41:15)
Feb 21 02:14:50 meshtest meshmonitor[16467]:     at process.processTicksAndRejections (node:internal/process/task_queues:103:5)
Feb 21 02:14:50 meshtest meshmonitor[16467]:     at async file:///opt/meshmonitor/node_modules/drizzle-orm/node-postgres/session.js:117:22
Feb 21 02:14:50 meshtest meshmonitor[16467]:     at async IgnoredNodesRepository.isNodeIgnoredAsync (file:///opt/meshmonitor/dist/db/repositories/ignoredNodes.js:154:26) {
Feb 21 02:14:50 meshtest meshmonitor[16467]:   query: 'select "nodeNum" from "ignored_nodes" where "ignored_nodes"."nodeNum" = $1',
Feb 21 02:14:50 meshtest meshmonitor[16467]:   params: [ 2733697270 ],
Feb 21 02:14:50 meshtest meshmonitor[16467]:   cause: error: value "2733697270" is out of range for type integer
Feb 21 02:14:50 meshtest meshmonitor[16467]:       at /opt/meshmonitor/node_modules/pg-pool/index.js:45:11
Feb 21 02:14:50 meshtest meshmonitor[16467]:       at process.processTicksAndRejections (node:internal/process/task_queues:103:5)
Feb 21 02:14:50 meshtest meshmonitor[16467]:       at async file:///opt/meshmonitor/node_modules/drizzle-orm/node-postgres/session.js:124:18
Feb 21 02:14:50 meshtest meshmonitor[16467]:       at async NodePgPreparedQuery.queryWithCache (file:///opt/meshmonitor/node_modules/drizzle-orm/pg-core/session.js:39:16)
Feb 21 02:14:50 meshtest meshmonitor[16467]:       at async file:///opt/meshmonitor/node_modules/drizzle-orm/node-postgres/session.js:117:22
Feb 21 02:14:50 meshtest meshmonitor[16467]:       at async IgnoredNodesRepository.isNodeIgnoredAsync (file:///opt/meshmonitor/dist/db/repositories/ignoredNodes.js:154:26) {
Feb 21 02:14:50 meshtest meshmonitor[16467]:     length: 148,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     severity: 'ERROR',
Feb 21 02:14:50 meshtest meshmonitor[16467]:     code: '22003',
Feb 21 02:14:50 meshtest meshmonitor[16467]:     detail: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     hint: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     position: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     internalPosition: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     internalQuery: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     where: "unnamed portal parameter $1 = '...'",
Feb 21 02:14:50 meshtest meshmonitor[16467]:     schema: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     table: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     column: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     dataType: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     constraint: undefined,
Feb 21 02:14:50 meshtest meshmonitor[16467]:     file: 'numutils.c',
Feb 21 02:14:50 meshtest meshmonitor[16467]:     line: '314',
Feb 21 02:14:50 meshtest meshmonitor[16467]:     routine: 'pg_strtoint32'
Feb 21 02:14:50 meshtest meshmonitor[16467]:   }
Feb 21 02:14:50 meshtest meshmonitor[16467]: }
```

**Screenshots**
n/a

**Environment (please complete the following information):**
* Host OS: Linux
* Deployment type: Proxmox LXC
* Any networking customizations: none
* Database backend: PostGreSQL

--- a/src/db/schema/ignoredNodes.ts
+++ b/src/db/schema/ignoredNodes.ts
@@ -10,13 +10,11 @@ import { sqliteTable, text, integer } from 'drizzle-orm/sqlite-core';
 import {
   pgTable,
   text as pgText,
-  integer as pgInteger,
   bigint as pgBigint,
 } from 'drizzle-orm/pg-core';
 import {
   mysqlTable,
   varchar as myVarchar,
-  int as myInt,
   bigint as myBigint,
 } from 'drizzle-orm/mysql-core';
 
@@ -34,7 +32,7 @@ export const ignoredNodesSqlite = sqliteTable('ignored_nodes', {
 // ============ IGNORED NODES (PostgreSQL) ============
 
 export const ignoredNodesPostgres = pgTable('ignored_nodes', {
-  nodeNum: pgInteger('nodeNum').primaryKey(),
+  nodeNum: pgBigint('nodeNum', { mode: 'number' }).primaryKey(),
   nodeId: pgText('nodeId').notNull(),
   longName: pgText('longName'),
   shortName: pgText('shortName'),
@@ -45,7 +43,7 @@ export const ignoredNodesPostgres = pgTable('ignored_nodes', {
 // ============ IGNORED NODES (MySQL) ============
 
 export const ignoredNodesMysql = mysqlTable('ignored_nodes', {
-  nodeNum: myInt('nodeNum').primaryKey(),
+  nodeNum: myBigint('nodeNum', { mode: 'number' }).primaryKey(),
   nodeId: myVarchar('nodeId', { length: 255 }).notNull(),
   longName: myVarchar('longName', { length: 255 }),
   shortName: myVarchar('shortName', { length: 255 }),

--- a/src/server/migrations/077_upgrade_ignored_nodes_nodenum_bigint.ts
+++ b/src/server/migrations/077_upgrade_ignored_nodes_nodenum_bigint.ts
@@ -1,0 +1,93 @@
+/**
+ * Migration 077: Upgrade ignored_nodes.nodeNum from INTEGER to BIGINT
+ *
+ * Meshtastic node numbers are unsigned 32-bit values (up to ~4.3 billion),
+ * but PostgreSQL/MySQL INTEGER is signed 32-bit (max 2,147,483,647).
+ * This causes query failures for nodeNum values above INT4 max.
+ *
+ * SQLite is unaffected because its INTEGER type is 64-bit.
+ *
+ * Fixes: https://github.com/Yeraze/meshmonitor/issues/1973
+ */
+
+import type { Database } from 'better-sqlite3';
+import { logger } from '../../utils/logger.js';
+
+/**
+ * SQLite migration: No changes needed
+ * SQLite INTEGER is already 64-bit
+ */
+export const migration = {
+  up: (_db: Database): void => {
+    logger.debug('Migration 077: SQLite INTEGER is already 64-bit, no changes needed');
+  },
+
+  down: (_db: Database): void => {
+    logger.debug('Migration 077 down: No changes needed for SQLite');
+  }
+};
+
+/**
+ * PostgreSQL migration: Upgrade ignored_nodes.nodeNum from INTEGER to BIGINT
+ */
+export async function runMigration077Postgres(client: import('pg').PoolClient): Promise<void> {
+  logger.info('Running migration 077 (PostgreSQL): Upgrading ignored_nodes.nodeNum to BIGINT...');
+
+  try {
+    const result = await client.query(`
+      SELECT data_type FROM information_schema.columns
+      WHERE table_name = 'ignored_nodes' AND column_name = 'nodeNum'
+    `);
+
+    if (result.rows.length === 0) {
+      logger.debug('Table ignored_nodes or column nodeNum does not exist, skipping');
+      return;
+    }
+
+    if (result.rows[0].data_type === 'bigint') {
+      logger.debug('ignored_nodes.nodeNum is already BIGINT, skipping');
+      return;
+    }
+
+    await client.query(`ALTER TABLE ignored_nodes ALTER COLUMN "nodeNum" TYPE BIGINT`);
+    logger.debug('Upgraded ignored_nodes.nodeNum to BIGINT');
+  } catch (error: any) {
+    logger.error('Failed to upgrade ignored_nodes.nodeNum:', error.message);
+    throw error;
+  }
+
+  logger.info('Migration 077 complete (PostgreSQL): ignored_nodes.nodeNum upgraded to BIGINT');
+}
+
+/**
+ * MySQL migration: Upgrade ignored_nodes.nodeNum from INT to BIGINT
+ */
+export async function runMigration077Mysql(pool: import('mysql2/promise').Pool): Promise<void> {
+  logger.info('Running migration 077 (MySQL): Upgrading ignored_nodes.nodeNum to BIGINT...');
+
+  try {
+    const [rows] = await pool.query(`
+      SELECT DATA_TYPE FROM information_schema.COLUMNS
+      WHERE TABLE_SCHEMA = DATABASE()
+        AND TABLE_NAME = 'ignored_nodes' AND COLUMN_NAME = 'nodeNum'
+    `);
+
+    if (!Array.isArray(rows) || rows.length === 0) {
+      logger.debug('Table ignored_nodes or column nodeNum does not exist, skipping');
+      return;
+    }
+
+    if ((rows[0] as any).DATA_TYPE === 'bigint') {
+      logger.debug('ignored_nodes.nodeNum is already BIGINT, skipping');
+      return;
+    }
+
+    await pool.query(`ALTER TABLE ignored_nodes MODIFY COLUMN nodeNum BIGINT PRIMARY KEY`);
+    logger.debug('Upgraded ignored_nodes.nodeNum to BIGINT');
+  } catch (error: any) {
+    logger.error('Failed to upgrade ignored_nodes.nodeNum:', error.message);
+    throw error;
+  }
+
+  logger.info('Migration 077 complete (MySQL): ignored_nodes.nodeNum upgraded to BIGINT');
+}

--- a/src/services/database.ts
+++ b/src/services/database.ts
@@ -82,6 +82,7 @@ import { migration as packetIdMigration, runMigration073Postgres, runMigration07
 import { migration as showMeshCoreNodesMigration, runMigration074Postgres, runMigration074Mysql } from '../server/migrations/074_add_show_meshcore_nodes_preference.js';
 import { migration as telemetryPacketIdBigintMigration, runMigration075Postgres, runMigration075Mysql } from '../server/migrations/075_upgrade_telemetry_packetid_bigint.js';
 import { migration as accuracyEstimatedPrefsMigration, runMigration076Postgres, runMigration076Mysql } from '../server/migrations/076_add_accuracy_and_estimated_position_prefs.js';
+import { migration as ignoredNodesNodeNumBigintMigration, runMigration077Postgres, runMigration077Mysql } from '../server/migrations/077_upgrade_ignored_nodes_nodenum_bigint.js';
 import { validateThemeDefinition as validateTheme } from '../utils/themeValidation.js';
 
 // Drizzle ORM imports for dual-database support
@@ -961,6 +962,7 @@ class DatabaseService {
     this.runShowMeshCoreNodesMigration();
     this.runTelemetryPacketIdBigintMigration();
     this.runAccuracyEstimatedPrefsMigration();
+    this.runIgnoredNodesNodeNumBigintMigration();
     this.ensureAutomationDefaults();
     this.warmupCaches();
     this.isInitialized = true;
@@ -2463,6 +2465,25 @@ class DatabaseService {
       logger.debug('accuracy_estimated_prefs migration completed successfully');
     } catch (error) {
       logger.error('Failed to run accuracy_estimated_prefs migration:', error);
+      throw error;
+    }
+  }
+
+  private runIgnoredNodesNodeNumBigintMigration(): void {
+    const migrationKey = 'migration_077_ignored_nodes_nodenum_bigint';
+    try {
+      const migrationStatus = this.getSetting(migrationKey);
+      if (migrationStatus === 'completed') {
+        logger.debug('Migration 077 (ignored_nodes nodeNum bigint) already completed');
+        return;
+      }
+
+      logger.debug('Running migration 077: Upgrade ignored_nodes.nodeNum to BIGINT...');
+      ignoredNodesNodeNumBigintMigration.up(this.db);
+      this.setSetting(migrationKey, 'completed');
+      logger.debug('ignored_nodes nodeNum bigint migration completed successfully');
+    } catch (error) {
+      logger.error('Failed to run ignored_nodes nodeNum bigint migration:', error);
       throw error;
     }
   }
@@ -10928,6 +10949,9 @@ class DatabaseService {
       // Run migration 076: Add show_accuracy_regions and show_estimated_positions
       await runMigration076Postgres(client);
 
+      // Run migration 077: Upgrade ignored_nodes.nodeNum to BIGINT
+      await runMigration077Postgres(client);
+
       // Verify all expected tables exist
       const result = await client.query(`
         SELECT table_name FROM information_schema.tables
@@ -11067,6 +11091,9 @@ class DatabaseService {
 
       // Run migration 076: Add show_accuracy_regions and show_estimated_positions
       await runMigration076Mysql(pool);
+
+      // Run migration 077: Upgrade ignored_nodes.nodeNum to BIGINT
+      await runMigration077Mysql(pool);
 
       // Verify all expected tables exist
       const [rows] = await connection.query(`

